app/src/main/java/com/tolkiana/tacoapp/TacoFriendsPagerAdapter.kt

```diff
@@ -1,13 +1,15 @@
 package com.tolkiana.tacoapp
 
+import android.content.res.Resources
 import android.support.v4.app.Fragment
 import android.support.v4.app.FragmentManager
 import android.support.v4.app.FragmentPagerAdapter
 
 /**
  * Created by tolkiana on 2/5/18.
  */
-class TacoFriendsPagerAdapter(fragmentManager: FragmentManager): FragmentPagerAdapter(fragmentManager) {
+class TacoFriendsPagerAdapter(private val resources: Resources, fragmentManager: FragmentManager): FragmentPagerAdapter(fragmentManager) {
```

Comment:
I often see the [`Context`](https://developer.android.com/reference/android/content/Context.html) class passed into classes instead of `Resources`. You could still call `context.getString()` in this file. I'm honestly not sure why I see that more though, especially if you're only using `Resources`, so don't feel the need to change this. I'm more just thinking out loud and will see if I can find an answer or best practice around this. 😄

Reply:
oh Interesting, maybe is one of the things we can discuss next session :)